testsuite/layers/pom.xml

```diff
@@ -2261,6 +2298,7 @@
                                                 <layer>jmx</layer>
                                                 <layer>jmx-remoting</layer>
                                                 <layer>jpa-distributed</layer>
+                                                <layer>jsf</layer>
```

Comment:
@yersan I think it's not strictly related to your MR but I don't understand the reason why on line [2263](https://github.com/wildfly/wildfly/pull/13330/files#diff-02180aff61cddbfaca58fc7682d79ab4R2263) the `artifactId` isn't parametrized like on line [1604](https://github.com/wildfly/wildfly/pull/13330/files#diff-02180aff61cddbfaca58fc7682d79ab4R1604):

```
<artifactId>wildfly-galleon-pack</artifactId>
```

Reply:
@tommaso-borgato If layers only available in wildfly-galleon-pack are part of what's provisioned then you can't use wildfly-ee-galleon-pack.